miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -37,8 +41,9 @@ internal class MiniAppWebView(
         settings.allowUniversalAccessFromFileURLs = true
         settings.domStorageEnabled = true
         settings.databaseEnabled = true
+        settings.setGeolocationEnabled(true)
```

Comment:
If we want to mandate the permission access via the bridge, would it make sense to set the geolocation flag to `false` as it defaults to `true`, what do you think @corycaywood @minh-rakuten?

Reply:
Ah, do you mean to set it to `false`?

Reply:
But if the host app is controlling the device permissions, we can't know when to change this value from `false` to `true`. Geolocation can't be accessed if this is set to `false` even if the device permission is granted, right?

Reply:
This value must be `true` to send the position data back to miniapp in case that miniapp uses geolocation request.